lib/kubeclient/config.rb

```diff
@@ -95,9 +99,7 @@ def ext_command_path(path)
       # - absolute (e.g. /path/to/foo)
       # - $PATH-based (e.g. curl)
       # - relative to config file's dir (e.g. ./foo)
-      if Pathname(path).absolute?
-        path
-      elsif File.basename(path) == path
+      if Pathname(path).absolute? || File.basename(path) == path
```

Comment:
Here I kinda liked the 3 separate branches mirroring the 3 cases listed in comment above, but not worth a rubocop:disable comment.